package.json

```diff
@@ -47,7 +47,7 @@
     "stream-browserify": "^3.0.0",
     "timers-browserify": "^2.0.12",
     "ts-loader": "^9.2.6",
-    "typescript": "^3.9.6",
+    "typescript": "4.4.4",
```

Comment:
Generally speaking I think you should separate out the new typescript/vscode deps into its own PR, or at least offer to. Regardless of whether this PR is accepted, having the version updates is a good idea. I expect that there is unlikely to be a hard version requirement on the older vscode/typescript in the project, but it may have one as Theia does not yet implement vscode 1.75.1 API